atom/browser/browser_win.cc

```diff
@@ -125,15 +125,47 @@ bool Browser::SetUserTasks(const std::vector<UserTask>& tasks) {
   return SUCCEEDED(destinations->CommitList());
 }
 
-bool Browser::RemoveAsDefaultProtocolClient(const std::string& protocol) {
-  if (protocol.empty())
-    return false;
+std::wstring protocolLaunchPath(std::string protocol, mate::Arguments* args) {
+  // Read in optional exe path arg
+  std::wstring exePath;
+  std::string rawExePath;
```

Comment:
Current `native_mate` doesn't have a converter for `std::wstring`.  Do you know of a way that I don't to bring in a `std::wstring` from the arguments?